rasterio/rio/edit.py

```diff
@@ -0,0 +1,65 @@
+"""Edit raster dataset metadata from the command line."""
+
+import json
+import logging
+
+import click
+
+import rasterio
+from rasterio.rio.cli import cli, file_in_arg
+from rasterio.transform import guard_transform
+
+
+@cli.command(short_help="Edit dataset metadata.")
+@file_in_arg
+@click.option('--nodata', type=float, default=None,
+              help="New nodata value")
```

Comment:
I tend to use single quotes around short names and key strings, double quotes around sentences or other longer texts. Reminds me that the latter ought to be grammatically correct as well as correct Python syntax, that's all.